tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3394,6 +3391,23 @@ std::tuple<int, int> GetTensorVectorSizeAndDim(
   return std::make_tuple(vector_size, vector_dim);
 }
 
+port::StatusOr<cudnn_frontend::Tensor> CreateCudnnTensor(
+    absl::Span<const int64_t> dims, absl::Span<const int64_t> strides,
+    int64_t uid, dnn::DataType dtype, int64_t vec_count, int64_t vec_dim,
+    bool is_virtual=false) {
+  auto tensor = cudnn_frontend::TensorBuilder()
+                    .setDim(dims.size(), dims.data())
+                    .setStrides(dims.size(), strides.data())
```

Comment:
`strides.size()`

If we want this function to take responsibility itself for ensuring setDims and setStrides are given the same length, it should check equality explicitly instead of silently truncating/out-of-bounds-accessing strides.

Reply:
Thanks for pointing this out. I changed it to `strides.size()`. PTAL.